appformer-form-modeler/appformer-form-modeler-codegen/appformer-form-modeler-codegen-impl/pom.xml

```diff
@@ -153,6 +153,21 @@
       <scope>test</scope>
       <type>test-jar</type>
     </dependency>
+    <dependency>
+      <groupId>org.kie.workbench.forms</groupId>
+      <artifactId>kie-wb-common-forms-fields</artifactId>
+      <version>7.0.0-SNAPSHOT</version>
```

Comment:
Actually, it seems that if you rebase off of master, you don't need to specify these versions as they are already managed for you :)